test/webWorker/initialize_test.js

```diff
@@ -1,10 +1,11 @@
 import webWorkerManager from '../../src/imageLoader/webWorkerManager.js';
+import { expect } from 'chai';
 
 describe('config', function () {
-  it('should initialize', function () {
+  it('should initialize and terminate', function () {
```

Comment:
Can we split these into two tests instead of asserting both conditions in a single test?

Reply:
okay, split into two tests